recipes/armadillo/all/test_package/CMakeLists.txt

```diff
@@ -1,12 +1,32 @@
 cmake_minimum_required(VERSION 3.15)
 project(PackageTest CXX)
 
-find_package(armadillo CONFIG REQUIRED)
+find_package(Armadillo CONFIG REQUIRED)
 if (LINK_HDF5)
     find_package(HDF5)
     set(HDF5_TARGETS HDF5::HDF5)
 endif()
 
 add_executable(example src/example.cpp)
-target_link_libraries(example armadillo::armadillo ${HDF5_TARGETS})
+target_link_libraries(example Armadillo::Armadillo ${HDF5_TARGETS})
```

Comment:
Same here:

```suggestion
target_link_libraries(example armadillo::armadillo ${HDF5_TARGETS})
```

Reply:
I don't really like it given that there's been a weak steer from the package maintainer that `Armadillo:Armadillo` is preferred, but since this isn't enforced I guess this does [match the CCI policy on CMake targets](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/test_packages.md#cmake-targets). I'll swap the primary target to `armadillo::armadillo` and make the aliases `Armadillo::Armadillo` and `armadillo`